---
bool left_pressed
bool right_pressed
float sonar
float light
